src/controllers/product.controller.js

```diff
@@ -0,0 +1,38 @@
+import Joi from "joi";
+import models from "../database/models";
+export const addProduct = async (req, res) => {
+  try {
+    const product = await models.Product.create(req.body);
+    res.status(201).json(product);
+  } catch (error) {
+    res.status(400).json({ message: error.message });
+  }
+};
+
+export const getAllProduct = async (req, res) => {
+  try {
+    const listProduct = await models.Product.findAll();
+    if (listProduct.length <= 0) {
+      res.status(404).json({
+        Status: "Not Found",
+        Message: "There is no product in Stock",
+      });
```

Comment:
Please anything that is not sucess, I mean status code of 2XX, send `error: your error message` instead `message: error message`